src/localisation/include/EKFNode.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Created By: Marcus Swift
+ * Created On: November 20th, 2017
+ * Description: An extended kalman filter node that takes in sensor data from
+ * the GPS, encoders and IMU and returns the bots estimated postion and
+ * orientation usng the ekf class
+ */
+
+#ifndef EKFNODE_EKFNODE_H
+#define EKFNODE_EKFNODE_H
```

Comment:
Just for conventions sake, this should be `PACKAGENAME_CLASSNAME_H` (so `LOCALISATION_EKFNODE_H`)

Reply:
changed